diff --git a/py/_builtin.py b/py/_builtin.py
index 52ee9d7..36991db 100644
--- a/py/_builtin.py
+++ b/py/_builtin.py
@@ -210,6 +210,7 @@ else:
 
     def exec_(obj, globals=None, locals=None):
         """ minimal backport of py3k exec statement. """
+        print("cooking exec: ", obj, globals, locals)
         __tracebackhide__ = True
         if globals is None:
             frame = sys._getframe(1)
